package.json

```diff
@@ -39,7 +39,7 @@
 		"@blueprintjs/select": "^3.12.0",
 		"@google-cloud/storage": "^5.8.1",
 		"@monaco-editor/react": "^4.1.1",
-		"@pubpub/prosemirror-pandoc": "^0.6.1",
+		"@pubpub/prosemirror-pandoc": "^0.7.0",
```

Comment:
Here are the bulk of the changes, for anyone interested: https://github.com/pubpub/prosemirror-pandoc/commit/f820a6977eb9704002766d550868bc708fbb23bf